app/controllers/students_learning_resources_controller.rb

```diff
@@ -25,4 +25,8 @@ def set_user
   def assignee_params
     params.permit(:user_id, :learning_resource_id)
   end
+
+  def my_students_ids
+    @user.group_members.map(&:user_id)
```

Comment:
`map` being an enum method will load and instantiate all `group_members` records. If you only care about `user_id`, then you might be using [pluck](https://apidock.com/rails/ActiveRecord/Calculations/pluck) instead

Reply:
Also I'm not sure if this is what you want, but looks like `@user` is a profesor and you want to get his/her students.
If that's the case, a nicer approach would be having a instance method responsible of getting that records. That way you can remove duplicity [here](https://github.com/bryanmg/ammdk_colima/pull/43/files#diff-24cbafec121d4311b5071b23b3cd02be297393a3a60dedb7c34173d435c4a627R55)
PLUS if `@user` actually represents a profesor, why don't we use the right var name?

e.g : `@profesor.students`

Reply:
I recently fixed the first comment, for the second one. I'll leave a TODO task.